aquality-automation/aquality-tracking-ui
# Feature/nunit3 add possibility to use fullname attribute as test name

# PR Details

Added possibility To choose test name node before importing NunitV3 file (Feature: Test Name will be used by default)

## Related Issue

#27 

## How Has This Been Tested

1. Import Nunit v3 file with Feature: Test Name option
1. See the imported test names
1. Import Nunit v3 file with Test Class Name option
1. See the imported test names

## Checklist
